packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -83,7 +135,14 @@ export default declare((api, options, dirname) => {
   }
 
   function hasMapping(methods, name) {
-    return has(methods, name) && (proposals || methods[name].stable);
+    return (
+      has(methods, name) &&
+      (proposals || methods[name].stable) &&
+      (polyfillPaths.has(methods[name].path) ||
+        methods[name]?.types?.some(type =>
```

Comment:
```suggestion
        methods[name].types?.some(type =>
```
nit: `methods.name` should not be nullish if `has(methods, name)` is `true`.

Reply:
True. Thanks!